mentions/src/main/kotlin/io/spine/examples/pingh/mentions/GitHubSearch.kt

```diff
@@ -35,17 +35,21 @@ import kotlin.jvm.Throws
 /**
  * Allows to access GitHub Search API.
  */
-public interface GitHubClientService {
+public interface GitHubSearch {
 
     /**
      * Searches for user `Mention`s on GitHub.
```

Comment:
From this description (including the following paragraphs) it is not clear whether we search for the mentions made by some user, or for the mentions made by someone else, which target the user.